<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>513</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ideas Organizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200,200,200);
background-color: rgb(10, 10, 10);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(239, 41, 41);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="styleSheet">
             <string notr="true">color: rgb(35,35,35);</string>
            </property>
            <property name="text">
             <string>Error</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_popup">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-radius: 5px;
}

QPushButton:hover {
	background-color: rgb(50, 50,50);
	color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="data_home">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(20, 20, 20);
border-radius:10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>301</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30);
	color: rgb(238, 238, 236);
}

QLineEdit:hover{
	border: 2px solid rgb(55,55,55);
	color: rgb(200,200,200);
}

QLineEdit:focus{
	border: 2px solid rgb(255,207,0);
}</string>
          </property>
          <property name="maxLength">
           <number>40</number>
          </property>
          <property name="placeholderText">
           <string>PROJECT NAME</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_language">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>100</y>
            <width>341</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30);
	color: rgb(238, 238, 236);
}

QLineEdit:hover{
	border: 2px solid rgb(55,55,55);
}

QLineEdit:focus{
	border: 2px solid rgb(255,207,0);
}</string>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="placeholderText">
           <string>PROGRAMMING LANGUAGE</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_date">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>190</y>
            <width>301</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30);
	color: rgb(238, 238, 236);
}

QLineEdit:hover{
	border: 2px solid rgb(55,55,55);
}

QLineEdit:focus{
	border: 2px solid rgb(255,207,0);
}</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
          <property name="placeholderText">
           <string>DATE IDEA ____/____ /____</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEdit_description">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>190</y>
            <width>341</width>
            <height>181</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit{
	border: 2px solid rgb(45,45,45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30);
	color: rgb(238, 238, 236);
}

QTextEdit:hover{
	border: 2px solid rgb(55,55,55);
}

QTextEdit:focus{
	border: 2px solid rgb(255,207,0);
}</string>
          </property>
          <property name="placeholderText">
           <string>DESCRIPTION</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_generate">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>310</y>
            <width>211</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
	color: rgb(238, 238, 236);
}

QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70,70,70);
	color: rgb(35,35,35);
}

QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255,165,24);
}</string>
          </property>
          <property name="text">
           <string>GENERATE TXT</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_title">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>-40</y>
            <width>411</width>
            <height>101</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>30</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(237, 212, 0);</string>
          </property>
          <property name="text">
           <string>
IDEAS ORGANIZER</string>
          </property>
         </widget>
         <zorder>label_title</zorder>
         <zorder>lineEdit_name</zorder>
         <zorder>lineEdit_language</zorder>
         <zorder>lineEdit_date</zorder>
         <zorder>textEdit_description</zorder>
         <zorder>pushButton_generate</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="footer">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(10, 10, 10);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(100,100,100);</string>
         </property>
         <property name="text">
          <string>Created By : Mateus Souza</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagens/file-rc.qrc"/>
 </resources>
 <connections/>
</ui>
